tests/post_training/data/wc_reference_data.yaml

```diff
@@ -1,22 +1,22 @@
 tinyllama_data_free_backend_OV:
   metric_value: 0.72057
-  num_int4: 228
+  num_int4: 114
   num_int8: 84
 tinyllama_data_aware_backend_OV:
   metric_value: 0.83853
-  num_int4: 188
+  num_int4: 94
   num_int8: 124
 tinyllama_data_aware_awq_stateful_backend_OV:
   metric_value: 0.85259
-  num_int4: 188
+  num_int4: 94
   num_int8: 124
 tinyllama_data_aware_awq_scale_estimation_backend_OV:
   metric_value: 0.83795
-  num_int4: 188
+  num_int4: 94
   num_int8: 124
 tinyllama_data_aware_awq_scale_estimation_stateful_backend_OV:
   metric_value: 0.83795
```

Comment:
please use the exact metric values that you've got in local and ci run: 0.84048

Reply:
updated